Crate fails to build with pin-project-lite v0.1.9

The crate fails to build with `pin-project-lite` at `v0.1.9` printing the following

```plain
error[E0453]: allow(explicit_outlives_requirements) overruled by outer forbid(rust_2018_idioms)
  --> /home/.cargo/registry/src/github.com-1ecc6299db9ec823/duplexify-1.2.1/src/lib.rs:42:1
   |
34 |   #![forbid(unsafe_code, rust_2018_idioms)]
   |                          ---------------- `forbid` level set here
...
42 | / pin_project_lite::pin_project! {
43 | |     /// Combine a reader + writer into a duplex of `Read` + `Write`.
44 | |     #[derive(Debug)]
45 | |     pub struct Duplex<R, W> {
...  |
50 | |     }
51 | | }
   | |_^ overruled by previous forbid
   |
   = note: this error originates in a macro (in Nightly builds, run with -Z macro-backtrace for more info)

error: aborting due to previous error

For more information about this error, try `rustc --explain E0453`.
error: could not compile `duplexify`.
```

See https://github.com/taiki-e/pin-project-lite/issues/34

--- a/README.md
+++ b/README.md
@@ -47,7 +47,7 @@ $ cargo add duplexify
 ```
 
 ## Safety
-This crate uses ``#![deny(unsafe_code)]`` to ensure everything is implemented in
+This crate uses ``#![forbid(unsafe_code)]`` to ensure everything is implemented in
 100% Safe Rust.
 
 ## Contributing

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -31,9 +31,14 @@
 //! # Ok(()) }) }
 //! ```
 
-#![forbid(unsafe_code, rust_2018_idioms)]
+#![forbid(unsafe_code)]
 #![deny(missing_debug_implementations, nonstandard_style)]
-#![warn(missing_docs, missing_doc_code_examples, unreachable_pub)]
+#![warn(
+    missing_docs,
+    missing_doc_code_examples,
+    unreachable_pub,
+    rust_2018_idioms
+)]
 
 use async_std::io::{self, BufRead, Read, Write};
 use async_std::task::{Context, Poll};

